### PR TITLE
Develop tristan

### PR DESCRIPTION
**Pull request by TristanBrattinga**
Added navbar and sidebar to layout
Added buildingcard popup

### CODE REVIEW

File: apps/web/src/lib/components/CardPopup.svelte [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
...jXH0ITGHpcFm/f9OC70/sNBbRwZBGn5hoceX7AH+AtwALM1gixdsQqYfs9Brk5q1N4OeNspfwv0F5NCoLF9Tzng4Y2NHyxDpdujGMxt5j+fV+2RGfW/E3SKUT9kDXEnB8xHbdQDTO7YRN1vq/FkOne3IOz60k9PIWuBNOWzMxZWWjR1CUr3S8lHkHW6j8+qcttaAzxEm0jmrbAXm57QzEwscNLafdIEWl5AuVt8kp1pZDG9GIoNctMWnPI/kCnrHVW37dcjO3+iI2xZkprvWkY4dDu3uRCp3PEB5B8MTeKwBEE02rgeucXjfg8hK3xCSnOky1uBG4EsO7xdxJBJ9vACZjfsIG58KvAXJVl5K+n75EZ5D2eZj/14uSlykjZWBacBnkcU0k81DSDyFV0JWtE4rk/HIkw7gt5htN22AWXN+ikaElku9WR+WI5ATPUz2n+u7IWVeLHmWyZ3I2YL5XOBf+m7EBYYGhJTlHu0uC8cj+ZBxfdBHtgypzNQo57p5N4E3TArkDpL7YonvBryL8n0RnO3V4nKxlOS+yLsKmokfGhpRpNzt2day0UJyXYZCjn+bhTmmrgjZT0FLoSVjO/F98iuXiuISNXYBX3CpKCdXI+vhVSMp3qHQDC5ToQWfso6KxumRHDB7V5EN6cLv8SZxshdJKasi7SSX4/u+S2WmXL2XkS1cH8GRSaxEQtWqyLkkL3g9U1RDRvMtivv1Ox3hExBT6pyPiqpGmpEkCt/Of4RqV+86i+T+2UPABbEu7LKITLKbgkKgSsp0zIE5twZr3QgL8VM/cACpSlpVOoAHSe6jISTFPjgX4n6peEWhFpSLk5Awd1Mf/T5UAxuxCnfOv6HgtpeFRciybpoK7P1I0W7n5F1oqSG1dj9pqf8epB5wo8/Mm5l8+/9TkIjkk5FYybRcA3zbS4ssaCJdKFOcrCd5bztkoeoyyRrc1FbywlTy5fh1Y86D0wEwQY6F60Ri19MatRV5DJqo+gC4H7enp3jlKNINgh7MhR4jqjoA6kiexoSb/8wieRD0kN75UM0B8DcCLfW6oovGq1nbyL67V5UBMIikzZ2XsX+c4GO/vRMxKCoauQU4h+yBHS8y+fYFepGKLDuRftmIrADuCtkoH7QjAR2b0Xp8lWUqE2gWqyiKoiiKoiiKUg3+B8WGthQSoFmmAAAAAElFTkSuQmCC" />
```
Comment (mtdvlpr) [inline apps/web/src/lib/components/CardPopup.svelte:50]: Maybe extract these image data strings to constants to clearify the code?

File: apps/web/src/lib/components/Map.svelte [hunk 2]
```diff
@@ -64,12 +81,15 @@
...
...221 23.0673 8.93267C24.6978 10.5631 25.7159 12.7061 25.95 15H25C24.7348 15 24.4804 15.1054 24.2929 15.2929C24.1054 15.4804 24 15.7348 24 16C24 16.2652 24.1054 16.5196 24.2929 16.7071C24.4804 16.8946 24.7348 17 25 17H25.95C25.7159 19.2939 24.6978 21.4369 23.0673 23.0673C21.4369 24.6978 19.2939 25.7159 17 25.95ZM16 11C15.0111 11 14.0444 11.2932 13.2221 11.8427C12.3999 12.3921 11.759 13.173 11.3806 14.0866C11.0022 15.0002 10.9031 16.0055 11.0961 16.9755C11.289 17.9454 11.7652 18.8363 12.4645 19.5355C13.1637 20.2348 14.0546 20.711 15.0245 20.9039C15.9945 21.0969 16.9998 20.9978 17.9134 20.6194C18.827 20.241 19.6079 19.6001 20.1573 18.7779C20.7068 17.9556 21 16.9889 21 16C21 14.6739 20.4732 13.4021 19.5355 12.4645C18.5979 11.5268 17.3261 11 16 11ZM16 19C15.4067 19 14.8266 18.8241 14.3333 18.4944C13.8399 18.1648 13.4554 17.6962 13.2284 17.1481C13.0013 16.5999 12.9419 15.9967 13.0576 15.4147C13.1734 14.8328 13.4591 14.2982 13.8787 13.8787C14.2982 13.4591 14.8328 13.1734 15.4147 13.0576C15.9967 12.9419 16.5999 13.0013 17.1481 13.2284C17.6962 13.4554 18.1648 13.8399 18.4944 14.3333C18.8241 14.8266 19 15.4067 19 16C19 16.7956 18.6839 17.5587 18.1213 18.1213C17.5587 18.6839 16.7956 19 16 19Z"
```
Comment (mtdvlpr) [inline apps/web/src/lib/components/Map.svelte:88]: Maybe make this SVG a component to improve readability?

File: apps/web/src/lib/components/Sidebar.svelte [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+		<span><Logo width="100px" />Amsterdam</span>
+		<button on:click={closeSidebar}>
+			<svg width="18" height="18" viewBox="0 0 18 18" fill="none" xmlns="http://www.w3.org/2000/svg">
+				<path fill-rule="evenodd" clip-rule="evenodd"
+					  d="M0.863563 0.156456C0.668301 -0.0388058 0.351719 -0.0388058 0.156457 0.156456C-0.0388056 0.351719 -0.0388059 0.668301 0.156456 0.863563L8.28831 8.99541L0.156576 17.1271C-0.0386864 17.3224 -0.0386863 17.639 0.156576 17.8342C0.351838 18.0295 0.66842 18.0295 0.863682 17.8342L8.99541 9.70252L17.127 17.8341C17.3223 18.0294 17.6389 18.0294 17.8341 17.8341C18.0294 17.6389 18.0294 17.3223 17.8341 17.127L9.70252 8.99541L17.8342 0.863685C18.0295 0.668423 18.0295 0.35184 17.8342 0.156578C17.639 -0.0386842 17.3224 -0.0386845 17.1271 0.156578L8.99541 8.2883L0.863563 0.156456Z"
+					  fill="#000" />
+			</svg>
+		</button>
+	</div>
+	<p>Filters</p>
```
Comment (mtdvlpr) [inline apps/web/src/lib/components/Sidebar.svelte:34]: Add these strings to the i18n config with English translations

File: apps/web/src/lib/styles/main.scss [hunk 4]
```diff
@@ -1,26 +1,36 @@
 @import './reset.scss';
 @import './variables.scss';
+@import "./typography.scss";
+
+html {
+  scroll-behavior: smooth;
+}
```
Comment (mtdvlpr) [inline apps/web/src/lib/styles/main.scss:7]: Is already present in the reset.scss

**Review by mtdvlpr — CHANGES_REQUESTED** [review]

File: apps/web/src/lib/components/Map.svelte (hunk 2 above)
Comment (TristanBrattinga) [reply apps/web/src/lib/components/Map.svelte:88]: Done! Heb een icons dir gemaakt waar de icons vandaan kunnen komen

File: apps/web/src/lib/components/Sidebar.svelte (hunk 3 above)
Comment (TristanBrattinga) [reply apps/web/src/lib/components/Sidebar.svelte:34]: Ik heb nu alleen ff het woord filters erin verwerkt. Laten we morgen samen ff de echte filters erin stoppen

File: apps/web/src/routes/[lang=lang]/(app)/map/+page.ts [hunk 5]
```diff
@@ -0,0 +1,19 @@
+import { i18nObject } from '$i18n/i18n-util';
+import type { MetaTagsProps } from 'svelte-meta-tags';
+
+export const load = async ({ parent, data: { buildings } }) => {
+	const { locale } = await parent();
+	const LL = i18nObject(locale);
+
+	const title = LL.filters();
```
Comment (mtdvlpr) [inline apps/web/src/routes/[lang=lang]/(app)/map/+page.ts:8]: This is the page title, so should be "map view" or something similar.

**Review by mtdvlpr — APPROVED** [review]
Nice work! One small thing and then it's good to go :)

**Pull request merged**